python-cellcutoff/setup.py

```diff
@@ -38,6 +38,20 @@ def get_version():
         return f.read().split('=')[-1].replace('\'', '').strip()
 
 
+def get_cxxflags():
+    """If the CXXFLAGS variable is defined (clang/osx) then get it."""
+    return os.environ.get("CXXFLAGS", "").split()
+
+
+def get_include_path():
+    """Get the conda include path from the prefix."""
+    prefix = os.environ.get("PREFIX", "")
+    if prefix:
+        return [os.path.join(prefix, "include")]
+    else:
+        return []
+
+
```

Comment:
I guess in principle setuptools should do this for us, but it may not work as it should.